CHANGELOG.md

```
@@ -343,6 +361,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 [Unreleased]: https://github.com/Noble-Lab/casanovo/compare/v5.1.2...HEAD
 [5.1.2]: https://github.com/Noble-Lab/casanovo/compare/v5.1.1...v5.1.2
+[Unreleased]: https://github.com/Noble-Lab/casanovo/compare/v5.1.1...HEAD
```

markdownlint-cli2 (0.21.0)

[warning] Link and image reference definitions should be needed
Duplicate link or image reference definition: "unreleased"

(MD053, link-image-reference-definitions)